kvbc/tools/object_store_utility/integrity_checker.cpp

```diff
@@ -75,15 +75,14 @@ std::pair<BlockId, Digest> IntegrityChecker::getLatestsCheckpointDescriptor() co
   LOG_INFO(logger_, "Last block: " << lastBlock);
   auto it = dynamic_cast<s3::Client*>(s3_dbset_.metadataDBClient.get())
                 ->getIterator<s3::Client::SortByModifiedDescIterator>();
-  auto [key, val] = it->seekAtLeast(Sliver::copy(checkpoints_prefix_.data(), checkpoints_prefix_.length()));
-  UNUSED(val);
+  const auto& kvdata = it->seekAtLeast(Sliver::copy(checkpoints_prefix_.data(), checkpoints_prefix_.length()));
   if (it->isEnd()) throw std::runtime_error("no checkpoints information in S3 storage");
 
-  std::string suff = key.toString().substr(checkpoints_prefix_.length());
+  std::string suff = kvdata.first.toString().substr(checkpoints_prefix_.length());
   BlockId block = util::to<BlockId>(suff.substr(0, suff.find_last_of('/')));
-  LOG_INFO(logger_, "Latest checkpoint descriptor: " << key.toString() << " for block: " << block);
+  LOG_INFO(logger_, "Latest checkpoint descriptor: " << kvdata.first.toString() << " for block: " << block);
```

Comment:
nit:
Use KVLOG macro wherever possible:
`LOG_INFO(logger_, "Latest checkpoint descriptor: " << kvdata.first.toString() << " for" << KVLOG(block);`

Reply:
Done.